rust/examples-frontend/svc-simulcast/src/index.ts

```diff
@@ -116,6 +118,11 @@ async function init()
         let newPreferredSpatialLayer: number;
         let newPreferredTemporalLayer: number;
 
+        if (isFirefox) {
+            setPreferredLayers(preferredSpatialLayer > 0 ? preferredSpatialLayer - 1 : spatialLayers - 1)
+            return
+        }
```

Comment:
I don't understand what this is and this will go into negative numbers as well unless I'm missing something.

Firefox supports both temporal and spatial layers just fine.